mapsforge-samples-android/src/main/java/org/mapsforge/samples/android/DownloadCustomLayerViewer.java

```diff
@@ -24,9 +24,7 @@
 public class DownloadCustomLayerViewer extends DownloadLayerViewer {
     @Override
     protected void createLayers() {
-        OnlineTileSource onlineTileSource = new OnlineTileSource(new String[]{
-                "a.tile.openstreetmap.fr", "b.tile.openstreetmap.fr", "c.tile.openstreetmap.fr"},
-                443);
+        OnlineTileSource onlineTileSource = new OnlineTileSource(new String[]{"tile.openstreetmap.fr"}, 443);
```

Comment:
This was not requested and is not working, have you tested it?

Reply:
In https://github.com/openstreetmap/operations/issues/737 it was requested to use the tile servers listed in the wiki only.

Didn't test it, I trusted the official documentation @ https://wiki.openstreetmap.org/wiki/Tile_servers to be correct - which it isn't. So "use the official sources" has to be taken with a grain of salt here :-(